kubernetes/adapter.go

```diff
@@ -300,10 +301,16 @@ func (a *Adapter) IngressFiltersString() string {
 func (a *Adapter) ListResources() ([]*Ingress, error) {
 	ings, err := a.ListIngress()
 	if err != nil {
+		log.Errorf("Failed to list ingress: %v", err)
```

Comment:
I would not log an error here since we return it on the line below and handle it from the caller.

Reply:
Ỳep duplication of logs, dropped